app/templates/layouts/asylum-base.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="charset" content="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>{% block page_title %}{{ config.APP_NAME }}{% endblock %}</title>
+
+        {% assets 'asylum_css' %}<link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}">{% endassets %}
+        {% assets 'asylum_scss' %}<link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}">{% endassets %}
+
+        <!-- Google Translate -->
+        <script src="http://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
+
+        <link href='https://fonts.googleapis.com/css?family=Open+Sans:600' rel='stylesheet' type='text/css'>
+
+        {% assets 'vendor_js' %}<script type="text/javascript" src="{{ ASSET_URL }}"></script>{% endassets %}
+        {% assets 'app_js' %}<script type="text/javascript" src="{{ ASSET_URL }}"></script>{% endassets %}
+        {% assets 'asylum_js' %}<script type="text/javascript" src="{{ ASSET_URL }}"></script>{% endassets %}
```

Comment:
Okay, so at a quick glance it looks like we shouldn't include the `link rel="alternate"` bit. In fact, I don't think their repo should have that either. It points to an rss feed for the content. BUT that feed is generated the default Jekyll way—which is to say based on the blog posts on their site. They have no blog posts, so the rss feed is empty (see asylumconnectcatalog.org/feed.xml). So we shouldn't have  that.

The `rel='canonical'` bit is more interesting. The canonical link tells Google/Bing/etc. that two pages are the same in indexing. So they use this in their repo to tell Google that asylumconnectcatalog.org/index.html is identical to asylumconnectcatalog.org/ (so Google doesn't index the same page twice). We don't have that problem since there is no `index.html` route in the flask app. However, if we set `rel="canonical"` then we CAN tell Google that `asylumconnectcatalog.org/philadelphia` is the same page as `asylumconnectcatalog.org/seattle` which is the same as the base page `asylumconnectcatalog.org/`. I am not sure whether or not that is a good idea.